src/main/scala/org/broadinstitute/hail/check/Gen.scala

```diff
@@ -20,6 +20,16 @@ case class Parameters(rng: RandomDataGenerator, size: Int, count: Int) {
 }
 
 object Gen {
+  def getRng(): Gen[RandomDataGenerator] = Gen { p => p.rng }
```

Comment:
In general, Scala seems to eschew the prefix `get`.  I'd be OK with just `def rng`.